src/main/java/gov/nasa/jpl/mbee/mdk/options/MDKOptionsGroup.java

```diff
@@ -26,7 +28,8 @@
             ENABLE_CHANGE_LISTENER_ID = "ENABLE_CHANGE_LISTENER_ID",
             ENABLE_COORDINATED_SYNC_ID = "ENABLE_COORDINATED_SYNC_ID",
             CUSTOM_USER_SCRIPT_DIRECTORIES_ID = "CUSTOM_USER_SCRIPT_DIRECTORIES_ID",
-            MMS_AUTHENTICATION_CHAIN = "MMS_AUTHENTICATION_CHAIN";
+            MMS_AUTHENTICATION_CHAIN = "MMS_AUTHENTICATION_CHAIN",
+    		DEFAULT_DOCBOOK_TO_PDF_STYLESHEET = "DEFAULT_DOCBOOK_TO_PDF_STYLESHEET";	
```

Comment:
Since this is user-configurable, i.e. changed from the default value, I think `DEFAULT_` should be omitted => `DOCKBOOK_TO_PDF_STYLESHEET`.

Reply:
Remove the Default.